Formula/z/zizmor.rb

```diff
@@ -0,0 +1,33 @@
+class Zizmor < Formula
+  desc "CLI tool for finding security issues in GitHub Actions setups"
+  homepage "https://crates.io/crates/zizmor"
```

Comment:
We should probably just use the GitHub page as the `homepage` to avoid `brew audit` failures.